driver/ppm_events.c

```diff
@@ -882,7 +902,11 @@ u16 pack_addr(struct sockaddr *usrsockaddr,
 {
 	u32 ip;
 	u16 port;
+#ifdef WDIG
+	ADDRESS_FAMILY family = usrsockaddr->sa_family;
+#else
 	sa_family_t family = usrsockaddr->sa_family;
+#endif
```

Comment:
A typedef in a header?

Reply:
This is a standard Windows define in ws2def.h